src/www/ui/api/Controllers/VersionController.php

```diff
@@ -42,8 +44,12 @@ class VersionController extends RestController
    */
   public function getVersion($request, $response, $args)
   {
-    $yamlDocArray = yaml_parse_file(__DIR__ .
-      "/../documentation/openapi.yaml");
+    try {
+      $yaml = new Parser();
+      $yamlDocArray = $yaml->parse(file_get_contents(__DIR__ ."/../documentation/openapi.yaml"));
```

Comment:
Won't it be good to use the syntax (`Yaml::parseFile(__DIR__ ."/../documentation/openapi.yaml")`) mentioned in the documentation of symfony/yaml instead of calling the underlying functions?

It will keep the code more readable and easy to maintain.

Reply:
Yes. 
for the version v3.3.18 the proposed syntax `(Yaml::parseFile(__DIR__ ."/../documentation/openapi.yaml"))` is not supported.
so i have chosen this way.

Reply:
Ah, I must have checked the latest documentation then.

Why not using the syntax for v3.3 (`Yaml::parse(file_get_contents('config.yml'));`)?

Reply:
Did you already check if it is working ??

Reply:
I don't this there is much logic which is confusing the code. apart from initializing the class. i think the code is fine.